### ISSUE TITLE
[CSS][A11y] Theme switch/toggle in top bar does not have accessible label

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/reactsocialnetwork#38]
**Describe the bug**

According to both Firefox console Accessibility test and Google Chrome Lighthouse, the Theme switch/toggle in top bar does not have accessible label.

**To Reproduce**

Run the dev server with `npm start` in the root dir. May need to `npm i` or `npm install `the app as I did not include my local node_modules dir. in the upload and it is in  the .gitignore.

**Expected behavior**
Screen reader accessibility

**Desktop (please complete the following information):**
   Linux, x64
   Firefox, Chrome
   both 107

**Additional context**

**Comment by pawel975** [comment]
I can work on it, you can assign me @gbowne1

**Comment by gbowne1** [comment]
Ok @pawel975 will do.

**Comment by pawel975** [comment]
I don't see any toggle theme switch a11y problem. I tested switch manually with screen reader and it says `toggle theme` as it is in `inputProps` param. I also generate Chrome lighthouse raport and it's fine.

**Comment by gbowne1** [comment]
Yeah I saw this too.  My goal is 98-100 across all tests.  I have not tested in a few days. 

Note: a lot of  the components have quite significant whitespace.  Mobile views should be next.. but more work needs done to the Nav's.  App.css has the media queries sizing I would like to start with.

**Comment by gbowne1** [comment]
I don't think it was talking about the screen reader though.  I tested in Firefox.  

There are a few issues with the toggle in the navbar itself.  

In the Firefox A11ly inspector test, hovering over the toggle reveals some things.
- checkbutton which says: Form elements must have a visible text label
- text container which says: Clickable elements must be focusable and should have interactive semantics

**Comment by gbowne1** [comment]
This issue may still exist, but with recent UI changes and moving to MUI based components and features.. closing this until more testing can be done and more features complete.  This will require more A11y testing with Lighthouse/Pagespeed, Firefox console A11y tool, etc.
